.github/workflows/ci.yml

```diff
@@ -14,6 +14,10 @@ jobs:
         uses: actions/checkout@v3
       - name: Install Node
         run: sudo apt-get install -y nodejs
+      - name: Install Bun
+      - uses: oven-sh/setup-bun@v1
```

Comment:
Just noticed the small mistake here. Fixed in https://github.com/rails/execjs/commit/d89ee741905c6b8dbbf7d9410287653b2f028dc7